content/docs/operating/configuration.md

```diff
@@ -472,15 +472,16 @@ all defined files are detected via disk watches and applied immediately. Files m
 provided in YAML or JSON format. Only changes resulting in well-formed target groups
 are applied.
 
-The JSON version of a target group has the following format:
-
+The JSON file must contain a list of target groups, using this format:
 ```
-{
-  "targets": [ "<host>", ... ],
-  "labels": {
-    [ "<labelname>": "<labelvalue>", ... ]
+[
+  {
+    "targets": [ "<host>", ... ],
+    "labels": {
+      [ "<labelname>": "<labelvalue>", ... ]
+    }
   }
```

Comment:
Maybe worth making it extra clear that more than on object can be specified here, something like this could be enough:

```json
  },
  ...
]
```